vrk-kpa/suomidigi
# SUOPA-137 Mobile menu

To test:
- `make fresh`
- Test the mobile menu and see that it works like it worked with mmenu library
